test/mocha.opts

```diff
@@ -2,6 +2,7 @@ test/helpers/*.js
 test
 --compilers js:babel-core/register
 --require ignore-styles
+--require babel-polyfill
```

Comment:
@wyattjoh It's already being transpiled because of the `--compilers` flag above, right? The tests don't just test server side code, they test client side code that use e.g. `import` which no version of node currently does.

I had to add this because that compiler is transpiling async functions to:
```
/Users/ben/dev/talk/test/graph/mutations/addCommentTag.js:20
  beforeEach(_asyncToGenerator(regeneratorRuntime.mark(function _callee() {
                               ^

ReferenceError: regeneratorRuntime is not def
```

So I would like to leave this in as it's unrelated to server side could requiring any form of transpiling. *All* the test code is already being transpiled and this just makes that transpilation not break things that *would* work if they weren't transpiled.

If you see this and still want me to remove this, say so and I will remove all use of async functions that necessitated it. But also... async functions are (IMO) great for readability compared to .thennables and now part of standard node 7.6.